from_mwt_ds/DataScience/vw_executor/loggers.py

```diff
@@ -109,16 +109,25 @@ class FileLogger(ILogger):
     def __init__(self,
                  path: Optional[Union[str, Path]],
                  level: str = 'INFO',
+                 reset: bool = False,
                  tag: str = '',
                  impl: Optional[_ILogger] = None):
         self.level_str = level
         if not impl:
-            Path(path).parent.mkdir(exist_ok=True, parents=True)
-            impl = _FileLoggerSafe(Path(path))
+            path = Path(path)
+            if reset and path.exists():
+                path.unlink()
+            path.parent.mkdir(exist_ok=True, parents=True)
+            impl = _FileLoggerSafe(path)
         super().__init__(impl, logging.getLevelName(self.level_str), tag)
 
     def __getitem__(self, key: str) -> 'FileLogger':
-        return FileLogger(path=None, level=self.level_str, tag = f'{self.tag}[{key}]', impl=self.impl)
+        return FileLogger(
+            path=None,
+            level=self.level_str,
+            reset=False,
```

Comment:
shouldn't this also be self.reset?

Reply:
we are not using it anywhere other than in constructor